examples/datasource-ciphered-password/README_pt.adoc

```diff
@@ -52,7 +52,7 @@ public static class ReverseEncryption implements PasswordCipher {
 Para ser funcional, ele precisa do arquivo
 `META-INF/org.apache.openejb.resource.jdbc.PasswordCipher/reverse`.
 
-O nome do arquivo (reversa) define o nome da criptografia a ser usado para o
+O nome do arquivo (reversa) define o nome da criptografia a ser usada para o
```

Comment:
Hi Daniel, I think "reverse" doesn't need to be translated, for better understanding

Reply:
Thanks for staying on top of this robinsonvs. I think that last one is not a blocker and we should merge it. OK?

Reply:
Ok thanks.